comtypes/tools/codegenerator.py

```diff
@@ -345,6 +346,11 @@ def ArrayType(self, tp):
     _enumvalues = 0
     def EnumValue(self, tp):
         value = int(tp.value)
+        if keyword.iskeyword(tp.name):
+            # XXX use logging!
+            if __debug__:
```

Comment:
I don't personally like the `__verbose__ = __debug__` pattern, because it has exactly the same problem I mentioned before - in practice, almost everyone will have verbose output enabled. But it's not my library, so if that's what they're doing...